### PR TITLE
#9095 Fix Mods page table filters (other than active)

### PR DESCRIPTION
**Pull request by BLoe**
## What does this PR do?

- Fixes the bug describe in #9095 

## Cursor Prompt

image

For more information on our expectations for the PR process, see the
[code review principles doc](https://www.notion.so/pixiebrix/Code-Review-Principles-1ce7276b82a84d2a995d55ad85e1310d?pvs=4)

### CODE REVIEW

File: src/extensionConsole/pages/mods/listView/ListView.tsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ const ListView: React.VoidFunctionComponent<ModsPageContentProps> = ({
   // Re-render the list when expandedRows changes.
   useEffect(() => {
     listRef.current?.resetAfterIndex(0);
-  }, [expandedRows]);
+  }, [expandedRows, listRef]);
```
Comment (grahamlangford) [inline src/extensionConsole/pages/mods/listView/ListView.tsx:54]: NIT: 
```suggestion
  }, [expandedRows, listRef.current]);
```

Comment (BLoe) [inline src/extensionConsole/pages/mods/listView/ListView.tsx:54]: This is unrelated to the bug, did not seem to change behavior. I just happened to notice the yellow squiggly line in my IDE and fixed it 🤷‍♂️

**Review by grahamlangford — APPROVED** [review]
